storage/postgres/postgres.go

```diff
@@ -108,6 +108,40 @@ func (p *Postgres) Close(ctx context.Context) error {
 	return nil
 }
 
+func (p *Postgres) ReadRelationshipTuples(
+	ctx context.Context,
+	filter storage.ReadRelationshipTuplesFilter,
+	opts ...storage.QueryOption,
+) (storage.TupleIterator, error) {
+	ctx, span := p.tracer.Start(ctx, "postgres.QueryRelationshipTuples")
+	defer span.End()
+
+	sql := "SELECT store, object_type, object_id, relation, _user, ulid, inserted_at FROM tuple WHERE store='%v'"
+	args := []interface{}{filter.StoreID}
+
+	if filter.OptionalObjectType != "" {
+		sql += " AND object_type='%v'"
```

Comment:
Use squirrel here? But this is nice! If we were not required to use positional arguments like `$1, $2, ...` and could just use `$, $, ...` then this would be as nice as using a query builder. We can't just use `$, $, ...` right? I suppose we could keep track of a counter, but then it would be a bit of a mess and, anyway, there is squirrel.